engine/src/main/java/org/terasology/rendering/dag/nodes/PrePostCompositeNode.java

```diff
@@ -56,17 +59,26 @@
  * [1] And refractions? To be verified.
  * [2] Currently not working: the code is there but it is never enabled.
  */
-public class PrePostCompositeNode extends AbstractNode {
+public class PrePostCompositeNode extends AbstractNode implements PropertyChangeListener {
     private static final ResourceUrn PRE_POST_MATERIAL = new ResourceUrn("engine:prog.prePostComposite");
 
     private RenderingConfig renderingConfig;
+    private WorldRenderer worldRenderer;
     private SubmersibleCamera activeCamera;
     private DisplayResolutionDependentFBOs displayResolutionDependentFBOs;
 
     private Material prePostMaterial;
 
-    @SuppressWarnings("FieldCanBeLocal")
-    private Vector4f skyInscatteringSettingsFrag = new Vector4f();
+    private boolean isLocalReflections;
+    private boolean isSsao;
+    private boolean isOutline;
+    private boolean isInscattering;
+    private boolean isVolumetricFog;
+
+    private StateChange setTexSceneReflectiveRefractiveNormals;
+    private StateChange setTexSsao;
+    private StateChange setTexEdges;
+    private StateChange setTexSceneSkyBand;
```

Comment:
Please rename the 10 variables above in line with:
- localReflectionsAreEnabled
- setReflectiveRefractiveNormalsInputTexture